src/System.Windows.Forms/src/System/Windows/Forms/ListView.cs

```diff
@@ -5461,6 +5546,11 @@ private unsafe IntPtr SendGroupMessage(ListViewGroup group, LVM msg, IntPtr lPar
                     break;
             }
 
+            if (group.TitleImageIndex != -1 || group.TitleImageKey != string.Empty)
```

Comment:
We have special consts for these:
```suggestion
            if (group.TitleImageIndex != ImageList.Indexer.DefaultIndex || group.TitleImageKey != ImageList.Indexer.DefaultKey)
```